packages/backend/src/graphql/resolvers/asset.ts

```diff
@@ -160,6 +160,31 @@ export const getAssetReceivingFee: AssetResolvers<ApolloContext>['receivingFee']
     return feeToGraphql(fee)
   }
 
+export const getFees: AssetResolvers<ApolloContext>['fees'] = async (
```

Comment:
Can we sort the fees by `createdAt` in a descending order? This way we will have latest fee at the top when showing the fees table.

Reply:
I also thought about this. But wasn't sure it was worth writing a new getPage function to do that. But I can.

Reply:
I think it makes sense to also display the payments and webhooks in descending order tbh. Can we pass a sorting option to the `getPage` function?

Reply:
Will do. Moving this out into a separate issue.